lib/config/typescriptReact.js

```diff
@@ -0,0 +1,23 @@
+var merge = require('merge');
+
+module.exports = {
+  extends: [
+    // react should come before typescript
+    'plugin:shopify/react',
+    'plugin:shopify/typescript',
+  ],
+
+  parserOptions: {
```

Comment:
Why are these needed?

Reply:
Oh yes, they aren't needed since the configs are extended.